docs/topics/proc_deploying-a-nodejs-application-to-openshift.adoc

```diff
@@ -0,0 +1,52 @@
+[id='deploying-a-node-js-application-to-openshift_{context}']
+= Deploying a {Node} application to OpenShift
+
+You can deploy a {Node} application to OpenShift using `nodeshift`.
+
+.Prerequisites
+
+* The `oc` CLI client installed.
+* `npm` installed.
+* Ensure all the ports used by your application are correctly exposed when configuring your routes.
+
+.Procedure
+
+. Log in to your OpenShift instance with the `oc` client.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc login ...
+----
+
+. Use `nodeshift` to deploy the application to OpenShift.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ npm run openshift
+----
+
+. Expose the application in OpenShift using link:https://docs.openshift.com/online/dev_guide/routes.html[a route].
```

Comment:
Is it possible to remove this whole section related to the routes and instead add the `--expose` flag to the "openshift" npm script?

I would also like to get away from using the `.nodeshift` directory if possible